cmd/observer/node_builder/observer_builder.go

```diff
@@ -0,0 +1,1018 @@
+package node_builder
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	consensus_follower "github.com/onflow/flow-go/module/upstream"
```

Comment:
can you merge the imports into 3 groups: stdlib, 3rd party (stuff outside for onflow/flow-go), and `onflow/flow-go`

Reply:
you'll need to `goimports --local`

Reply:
Addressed: https://github.com/GetElastech/flow-go/commit/8ff48be52f60861ac00f32ec5f62092386e8edf3